### DISCUSSION TITLE
Updraft Section 4 Lesson 42 - Written lesson has mixed up content

### DISCUSSION THREAD

**Discussion by rube-de** [Cyfrin/security-and-auditing-full-course-s23#104] [General]
The written less of Lesson 42 has mixed up content and a broken link to a picture. 
(See below)

https://updraft.cyfrin.io/courses/security/puppy-raffle/test-coverage?lesson_format=transcript

![image](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/8930910/9909f6a9-1d4d-4804-83d8-ce11f9660208)

**Comment by PatrickAlphaC** [comment]
Thanks! Will fix

**Comment by cromewar** [comment]
Solved on the next push
